### DISCUSSION TITLE
Why Datapane automatically converting d-m-Y to ISO format

### DISCUSSION THREAD

**Discussion by ehsen** [datapane/datapane#50] [Help]
I have a pandas dataframe with date column in 'd-m-Y' format, the date column is string not datetime. But when I create datapane report it automatically converts my date to ISO format. Similarly I have another column with int datatype, datapane also converts it into ISO date (which is very weird).

Can you please tell me how can I fix this.

**Comment by lanthias** [comment]
Thanks a lot for the note, I've converted this to [an issue](https://github.com/datapane/datapane/issues/58) so we can track it there.
